### DISCUSSION TITLE
ARM构型的处理器可以食用吗？

### DISCUSSION THREAD

**Discussion by stars-spark** [koishijs/koishi#936] [Q&A]
### Describe the problem related to the feature request

比如树莓派和jetson开发板运行

### Describe the solution you'd like

发布适合arm的.deb

**Comment by ilharp** [comment]
可以直接使用 [创建模板项目](https://koishi.chat/manual/starter/boilerplate.html) 方式创建 Koishi 项目。

**Comment by stars-spark** [comment]
> 可以直接使用 [创建模板项目](https://koishi.chat/manual/starter/boilerplate.html) 方式创建 Koishi 项目。
> 

![IMG_20230117_202330.jpg](https://user-images.githubusercontent.com/53867496/212898281-81dc0c5b-0063-4c45-ae79-76c610ac843b.jpg)![IMG_20230117_202338.jpg](https://user-images.githubusercontent.com/53867496/212898301-213cb6f3-a6da-4384-80a8-a811c205c6e2.jpg)执行这个命令时卡了很久

**Comment by stars-spark** [comment]
> > 可以直接使用 [创建模板项目](https://koishi.chat/manual/starter/boilerplate.html) 方式创建 Koishi 项目。
> > 
> 
> ![IMG_20230117_202330.jpg](https://user-images.githubusercontent.com/53867496/212898281-81dc0c5b-0063-4c45-ae79-76c610ac843b.jpg)![IMG_20230117_202338.jpg](https://user-images.githubusercontent.com/53867496/212898301-213cb6f3-a6da-4384-80a8-a811c205c6e2.jpg)执行这个命令时卡了很久
npm init koishi

**Comment by MirrorCY** [comment]
> > > 可以直接使用 [创建模板项目](https://koishi.chat/manual/starter/boilerplate.html) 方式创建 Koishi 项目。
> > 
> > 
> > ![IMG_20230117_202330.jpg](https://user-images.githubusercontent.com/53867496/212898281-81dc0c5b-0063-4c45-ae79-76c610ac843b.jpg)![IMG_20230117_202338.jpg](https://user-images.githubusercontent.com/53867496/212898301-213cb6f3-a6da-4384-80a8-a811c205c6e2.jpg)执行这个命令时卡了很久
> > npm init koishi

https://koishi.chat/manual/starter/boilerplate.html#%E9%85%8D%E7%BD%AE%E9%95%9C%E5%83%8F%E6%BA%90
如果你是国内用户，从 npm 或 yarn 上下载依赖可能非常慢。因此，我们推荐你配置一下镜像源，以提升安装速度。

**Comment by ilharp** [comment]
你可能误解了一些事情——你应当可以看到 `added 427 packages in 45s` 的标识。这表明 Koishi 在 45 秒的时间内已经安装完成并已经启动，只是由于你未配置浏览器或浏览器损坏导致 Koishi 控制台没有自动弹出。现在你可以使用浏览器打开 `http://localhost:5140` 以打开你的 Koishi 控制台，并开始使用。

---

注：楼上的解决方法是错误的。你在 45 秒后就已完成安装并已启动了 Koishi。

**Comment by MirrorCY** [comment]
嗷，还真是，拍摄屏幕使我思维紊乱

**Comment by MaikoTan** [comment]
lol, you two just funny.

**Comment by stars-spark** [comment]
emm，安装好对应的deb包后，找不到安装位置在哪里，搜文件名，里面没有对应文件![IMG_20230117_214852.jpg](https://user-images.githubusercontent.com/53867496/212915910-8afcc52c-2970-4b1f-9c63-1a70ef9c4783.jpg)![IMG_20230117_214850.jpg](https://user-images.githubusercontent.com/53867496/212915928-dea67f69-4876-4fc9-abb0-06ae81bf3fd8.jpg)

**Comment by stars-spark** [comment]
等等是不是要下压缩包而不是deb

**Comment by HuanLinOTO** [comment]
试试运行 ```gocqhttp``` 命令或者 ```go-cqhttp```

**Comment by ilharp** [comment]
Koishi 没有提供 deb 安装方式，目前，只能使用 [创建模板项目](https://koishi.chat/manual/starter/boilerplate.html) 方式创建 Koishi 项目。

---

注：楼上的解决方法是错误的。Koishi 没有提供 deb 安装方式。

**Comment by HuanLinOTO** [comment]
> Koishi 没有提供 deb 安装方式，目前，只能使用 [创建模板项目](https://koishi.chat/manual/starter/boilerplate.html) 方式创建 Koishi 项目。
> 
> 注：楼上的解决方法是错误的。Koishi 没有提供 deb 安装方式。

他好像是在问 gocqhttp 使用 deb 方式安装后怎么启动

**Comment by ilharp** [comment]
> 他好像是在问 gocqhttp 使用 deb 方式安装后怎么启动

本仓库为 Koishi 的官方仓库。如需询问 gocqhttp 相关问题，请前往对应仓库询问。

**Comment by ilharp** [comment]
另外需要注意，如果你正在接入 QQ，那么应当查看最新版本的 [文字教程](https://k.ilharp.cc/r3) 或 [视频教程](https://b23.tv/BV1W14y137rt)。官方文档中的方法已过时。
